guides/common/modules/ref_network-interface-configuration-options.adoc

```diff
@@ -0,0 +1,38 @@
+[id="network_interface_configuration_options"]
+= Network interface configuration options
+
+When adding a network interface, you need to specify several configuration options. The following lists provide information on the relevant options for the different types of interfaces.
+
+.Physical interface settings
+
+*Device Identifier*:: The identifier is used to specify this physical interface when creating bonded interfaces, VLANs, and aliases.
+
+*DNS name*:: {Project} saves this name in the {SmartProxyServer} associated with the selected domain (the "DNS A" field) and {SmartProxyServer} associated with the selected subnet (the "DNS PTR" field).
+A single host can therefore have several DNS entries.
+
+*Domain*:: Specifies the domain associated with the network interface. To create and manage domains, navigate to *Infrastructure* > *Domains*.
+
+*Subnet*:: Specifies the subnet associated with the network interface. To create and manage subnets, navigate to *Infrastructure* > *Subnets*.
+
+*Managed* interface:: If the interface is managed, configuration is pulled from the associated {SmartProxyServer} during provisioning, and DNS and DHCP entries are created.
+If using Kickstart provisioning, a configuration file is automatically created for the interface.
+
+*Primary* interface:: The DNS name from the primary interface is used as the host portion of the FQDN.
+
+*Provision*:: TFTP boot takes place using the provisioning interface.
```

Comment:
```suggestion
*Provision*:: Network boot uses the provisioning interface.
```

Reply:
@stejskalleos Is is this change acceptable?

Reply:
LGTM